test/views/DeclarationInformationViewSpec.scala

```diff
@@ -84,6 +87,23 @@ class DeclarationInformationViewSpec extends UnitViewSpec with Injector {
       view.select(".submission__mrn .govuk-summary-list__value").first().text() mustBe submission.mrn.get
     }
 
+    "contains create EAD link" in {
+
+      val generateEADLink = view.getElementById("generate-ead")
+
+      generateEADLink.text() mustBe "submissions.generateEAD"
+      generateEADLink must haveHref(controllers.pdf.routes.EADController.generatePdf(submission.mrn.get))
+    }
+
+    "doesn't contains EAD link if there is no MRN" in {
+
+      val view = declarationInformationPage(submission(None), notifications)(request, messages)
+
+      intercept[NullPointerException] {
+        view.getElementById("generate-ead").text()
+      }
```

Comment:
Isn't there a better way to chech for absence? Not checking through exceptions?

Reply:
I don't know a better way. I used this, because the `mustBe empty` didn't work. If you have any ideas please let me know if you know how to do it in better way